clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRebalanceListener.java

```diff
@@ -78,6 +78,14 @@
  */
 public interface ConsumerRebalanceListener {
 
+    ConsumerRebalanceListener NO_OP = new ConsumerRebalanceListener() {
```

Comment:
This should be static.

Reply:
I think `static` is not required inside the interface.